test/NuGet.Tests.Apex/NuGet.Tests.Apex/NuGetPackageSigningTests/AuthorSignedPackageTestCase.cs

```diff
@@ -260,8 +260,7 @@ public AuthorSignedPackageTestCase(SignedPackagesTestsApexFixture apexSigningFix
                 testContext.Project.Build();
                 testContext.NuGetApexTestService.WaitForAutoRestore();
 
-                // TODO: Fix bug where no warnings are shown when package is untrusted but still installed
-                //nugetConsole.IsMessageFoundInPMC("expired certificate").Should().BeTrue("expired certificate warning");
+                nugetConsole.IsMessageFoundInPMC("NU3011: The author primary signature validity period has expired.").Should().BeTrue("expired certificate warning");
```

Comment:
while i won't block this PR on this comment, but these tests will break when you change the error string for NU3011. Instead you might want to make the resources.resx which contains the above string public/internal and access it from here.